drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -347,15 +347,190 @@ int wifi_nrf_get_power_save_config(const struct device *dev,
 	return ret;
 }
 
+/* TWT interval conversion helpers: User <-> Protocol */
+static struct twt_interval_float wifi_nrf_twt_ms_to_float(uint32_t twt_interval_ms)
```

Comment:
(Sorry missed this in my earlier review) Instead of having a structure as a return value, it would be better to pass a pointer to a structure variable as a parameter.

Reply:
Why? IMHO, this is a small struct so it should be fine.